### ISSUE TITLE
Ability to make virtual devices permanent?

### ISSUE THREAD

**Issue by ToxMox** [eiz/SynchronousAudioRouter#26]
I absolutely love SAR but my biggest issue with it is that the virtual audio devices are created on the fly when I open Reaper. If I close Reaper for any reason the devices go away. This is problematic for me because things like Discord and OBS will reset all my custom selected SAR audio devices back to Default when that happens if they happen to be running when Reaper isn't. I wish there was a way to make the SAR devices just stay there.

**Comment by cliffrowley** [comment]
Indeed. Not only does it make configuration of other applications a pain, but also means that they often crash if I restart the DAW. There also seems to be no guarantee that they'll come back in the right order either, so I am regularly having to re-configure all my port mappings etc. It's getting tedious.
